app/containers/Toast.js

```diff
@@ -20,15 +19,17 @@ const styles = StyleSheet.create({
 	}
 });
 
-export const LISTENER = 'Toast';
-
 class Toast extends React.Component {
 	static propTypes = {
 		theme: PropTypes.string
 	}
 
+	static mounted = null;
+
+	static show = (...args) => Toast.mounted && Toast.mounted.showToast(...args);
```

Comment:
Instead of static, can we use refs instead? Maybe even migrate to hooks.

Reply:
Looking into it!